src/xenia/kernel/xam/user_profile.cc

```diff
@@ -18,8 +18,8 @@ namespace kernel {
 namespace xam {
 
 UserProfile::UserProfile() {
-  xuid_ = 0xBABEBABEBABEBABE;
-  name_ = "User";
+  xuid_ = 0xE000BABEBABEBABE;
```

Comment:
This change isn't related to the PR right?

Reply:
oof no, clicked the wrong button